docs/tutorials/infer_phys_pars.rst

```diff
@@ -627,43 +627,52 @@ Pulsar orbital inclination
 
 Next, the relation between pulsar distance and orbital inclination can be
 solved for :math:`\sin( i_\mathrm{p} )`. This relation first needs to be
-rewritten as a (somewhat ugly) quadratic equation in
+rewritten as a quadratic equation in
 :math:`\sin^2( i_\mathrm{p} )`:
 
 .. math::
 
     \cos^2( \chi_\mathrm{p} ) \sin^4( i_\mathrm{p} )
         - ( 1 + Z^2 ) \sin^2( i_\mathrm{p} ) + Z^2 = 0,
     \qquad \mathrm{with} \qquad
-    Z = \frac{ v_{0,\oplus} K_\mathrm{p} b_\oplus }
+    Z = \frac{ \sin( i_\mathrm{p} ) }{ b_\mathrm{p} }
+      = \frac{ v_{0,\oplus} K_\mathrm{p} b_\oplus }
              { A_\oplus A_\mathrm{p} d_\mathrm{p} }.
 
-The standard quadratic formula then gives the solutions
+Here, the auxiliary parameter :math:`Z` is restricted to the range
```

Comment:
Is it really useful to discuss the range in `Z` with `cos(\chi)=1` special-cased?

Reply:
Removed. Indeed this was more something that came up analyzing the properties of that equation, and it's not very useful for the tutorial.